doc/howto/network_acls.md

```diff
@@ -136,6 +136,16 @@ For example:
 source=@internal
 ```
 
+If your network supports [network peers](network_ovn_routing.md), you can reference traffic to or from the peer connection by using a network subject selector in the format `@<network_name>/<peer_name>`.
```

Comment:
Can we call the `network_ovn_routing.md` file `network_ovn_peers.md` instead as that is what the feature is called.